sound/sequences/00_sound_player.s

```diff
@@ -2097,11 +2097,11 @@ chan_setlayer 0, .layer_C7C
 chan_end
 
 .layer_C7C:
-layer_transpose -2
+layer_transpose_voice -2
 layer_somethingon
-layer_portamento 0x85, 39, 255
-layer_note1 42, 0x1e, 110
-layer_note1 39, 0x41, 110
+layer_portamento_voice 0x85, 39, 255
+layer_note1_voice 42, 0x1e, 110
+layer_note1_voice 39, 0x41, 110
```

Comment:
there are two notes here, these should likely be made one

Reply:
`layer_somethingon` takes care of this automatically; it'll sound fine in-game.